secant/Dependencies/SDKSynchronizer/SDKSynchronizerInterface.swift

```diff
@@ -18,24 +18,11 @@ extension DependencyValues {
 }
 
 enum SDKSynchronizerState: Equatable {
-    case unknown
-    case transactionsUpdated
-    case started
     case progressUpdated
-    case statusWillUpdate
-    case synced
+    case started
     case stopped
-    case disconnected
-    case syncing
```

Comment:
Why is this state deleted?

Reply:
the interface uses it's own state for the secant needs, copy paste of all states in the past didn't make sense anymore